### PR TITLE
Relicense project to Apache-2.0 license

### PR DESCRIPTION
**Pull request by n-eiling**
This make the project more compatible to other permissively licensed projects.

Signed-off-by: Niklas Eiling <niklas.eiling@eonerc.rwth-aachen.de>

### CODE REVIEW

**Comment by n-eiling** [comment]
I approve this motion to relicense VILLASfpga under the Apache 2.0 license.

**Review by stv0g — CHANGES_REQUESTED** [review]
For consistency I would prefer if we could keep the same header style as in VILLASnode & common:

https://github.com/VILLASframework/node/blob/master/include/villas/api.hpp

Also in VILLASnode/common we include the license as Markdown file named COPYING.md.

If you dislike anything about the existing style, I am also okay changing it.
However, only if we maintain project wide consistency.
Meaning we need similar PRs for VILLASnode and common.

**Review by daniel-k — APPROVED** [review]
I approve this motion to relicense VILLASfpga under the Apache 2.0 license.

**Comment by hatimak** [comment]
I approve this motion to relicense VILLASfpga under the Apache 2.0 license.

**Comment by n-eiling** [comment]
Thanks a lot to everyone! We now have the approvement of all authors. @stv0g do you want to change the patch or should we just rebase and merge the current version?

**Comment by stv0g** [comment]
Let me make some modifications. I am planning on removing the year range from the file headers copyright notice as this is just a lot of noise if we need to update them every year. There are several other big Open Source projects which are doing the same lately. I would like to keep only they year when the file was first created or mostly rewritten. That way we never need to update them again..

**Comment by stv0g** [comment]
I am planning on applying this to all VILLAS components

**Comment by n-eiling** [comment]
What do you think about this: https://reuse.software/dev/

**Comment by stv0g** [comment]
> What do you think about this: https://reuse.software/dev/

Sounds good! Lets use it :)

**Comment by stv0g** [comment]
@n-eiling I started from scratch an fixed all the unclear license/copyright notices. The project is now REUSE compliant and also checked in the future for its compliancy via the GitLab CI

File: LICENSES/BSD-3-Clause.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Copyright (c) <year> <owner>.
```
Comment (n-eiling) [inline LICENSES/BSD-3-Clause.txt:1]: is this correct?

Comment (stv0g) [reply LICENSES/BSD-3-Clause.txt:1]: It was added like this be the REUSE tool as it automatically downloads and puts the licenses into this folder.

But I guess we should adapt it by hand then... Let me push the change.

Comment (stv0g) [reply LICENSES/BSD-3-Clause.txt]: Fixed

**Pull request merged**